types/mongo-sanitize/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for mongo-sanitize 1.0
+// Project: https://github.com/vkarpov15/mongo-sanitize
+// Definitions by: Cedric Cazin <https://github.com/CedricCazin>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+export function sanitize(v: any): any;
```

Comment:
While technically the argument can be anything, [it seems like the intent is only to pass objects](https://github.com/vkarpov15/mongo-sanitize/blob/5a1b071191f7bb74410e2f8d7aca2c7b2d4b3164/index.js#L2). So perhaps you want to use `object` instead?

I was also thinking this should potentially be

```ts
export function sanitize<T extends object>(v: T): T;
```

for most cases users actually care about. I'll await your thoughts and/or changes before merging in.